src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -278,6 +278,16 @@ public static bool IsWindowsElasticPremium(this IEnvironment environment)
             return string.Equals(value, ScriptConstants.ElasticPremiumSku, StringComparison.OrdinalIgnoreCase);
         }
 
+        /// <summary>
+        /// Gets a value indicating whether the application is running in a Windows App Service environment.
+        /// </summary>
+        /// <param name="environment">The environment to verify.</param>
+        /// <returns><see cref="true"/> if running in a Windows App Service app; otherwise, false.</returns>
+        public static bool IsAnyWindows(this IEnvironment environment)
```

Comment:
The naming is currently vague; I'd prefer if we can make it clear this is for a _hosted_ windows environment. Side note: had this issue with our naming of `IsAnyKubernetes()` just last week. Was it really _any_ k8s? Even self hosted k8s? Turns out no, it is specific hosted k8s only.

```suggestion
        public static bool IsAnyHostedWindows(this IEnvironment environment)
```